Apps/Playground/iOS/LibNativeBridge.mm

```diff
@@ -66,6 +68,7 @@ - (void)init:(void*)view width:(int)inWidth height:(int)inHeight
 
 - (void)resize:(int)inWidth height:(int)inHeight
 {
+    graphics->UpdateSize(static_cast<size_t>(inWidth), static_cast<size_t>(inHeight));
```

Comment:
Given the order of operations for iOS, I don't think it should be possible for either of them to be null here. (It seems impossible that resize should be called before init.) I don't know that for sure, though; and since that isn't the point of this PR, I can just copy the existing pattern for now.